src/executables/compactdb_common.c

```diff
@@ -255,7 +255,7 @@ get_class_mops_from_file (const char *input_filename, MOP ** class_list, int *nu
 	  goto end;
 	}
 
-      strncpy (class_names[i], buffer, len);
+      strncpy (class_names[i], buffer, len + 1);
```

Comment:
Is there a reason to copy with a size of "len+1"?
If you decide to modify it anyway, I recommend doing it as follows.

memcpy (class_names[i], buffer, len);

Reply:
+1 is necessary because class_names[i] must end with a terminating zero. Isn't it?
Note that len is strlen(buffer).

Reply:
@hyunikn
In the code, '\0' is filled with the substitution statement immediately below.
`class_names[i][len] = 0;`